### ISSUE TITLE
US-ASCII error on default grok-pattern file

### ISSUE THREAD

**Issue by Kuckkuck** [fluent/fluent-plugin-grok-parser#32]
Hi,

since fluent version 0.14.17/0.14.18 with the fluent-plugin-grok-parser plugin 2.3.1, which is reading the default grok-patterns file, I get a /var/lib/gems/2.3.0/gems/fluent-plugin-grok-parser-2.1.3/lib/fluent/plugin/grok.rb:39:in `block in add_patterns_from_file': invalid byte sequence in US-ASCII (ArgumentError).
The default grok pattern file contains month where the german character "ä" ( May/März) is not us-ascii compatible. This was working with 0.14.16 with the same plugin version for fluent-plugin-grok-parser 2.3.1., but now it is crashing. I am using an Ubuntu container, where I have just changed the version number for fluentd and now it is crashing with: block in add_patterns_from_file': invalid byte sequence in US-ASCII (ArgumentError).

```
2017-06-30 09:57:27 +0000 [info]: reading config file path="/etc/fluent/fluent.conf"
/var/lib/gems/2.3.0/gems/fluent-plugin-grok-parser-2.1.3/lib/fluent/plugin/grok.rb:39:in `block in add_patterns_from_file': invalid byte sequence in US-ASCII (ArgumentError)
from /var/lib/gems/2.3.0/gems/fluent-plugin-grok-parser-2.1.3/lib/fluent/plugin/grok.rb:38:in each_line'
from /var/lib/gems/2.3.0/gems/fluent-plugin-grok-parser-2.1.3/lib/fluent/plugin/grok.rb:38:in add_patterns_from_file'
from /var/lib/gems/2.3.0/gems/fluent-plugin-grok-parser-2.1.3/lib/fluent/plugin/parser_grok.rb:30:in block in configure'
from /var/lib/gems/2.3.0/gems/fluent-plugin-grok-parser-2.1.3/lib/fluent/plugin/parser_grok.rb:29:in glob'
from /var/lib/gems/2.3.0/gems/fluent-plugin-grok-parser-2.1.3/lib/fluent/plugin/parser_grok.rb:29:in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/plugin.rb:164:in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/plugin_helper/parser.rb:90:in block in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/plugin_helper/parser.rb:85:in each'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/plugin_helper/parser.rb:85:in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/plugin/filter_parser.rb:42:in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/plugin.rb:164:in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/agent.rb:152:in add_filter'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/agent.rb:70:in block in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/agent.rb:64:in each'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/agent.rb:64:in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/root_agent.rb:112:in configure'
from /var/lib/gems/2.3.0/gems/fluentd-0.14.18/lib/fluent/engine.rb:131:in configure'
```

Any idea, what was changed?

Regards,

Olaf

**Comment by okkez** [comment]
Same as #27 ???

**Comment by okkez** [comment]
I will check this issue later.

**Comment by okkez** [comment]
Related to https://github.com/fluent/fluentd/issues/1606 ?

**Comment by okkez** [comment]
Released 2.1.4

**Comment by Kuckkuck** [comment]
great, thanks it worked.
